tjmonopix2/analysis/plotting.py

```diff
@@ -343,8 +353,8 @@ def create_threshold_map(self):
                 z_min = 0
                 z_max = 16
             else:
-                electron_axis = False  # TODO: True after calibration code is done
-                use_electron_offset = False  # TODO: True after calibration code is done
+                electron_axis = self.electron_axis
+                use_electron_offset = False
```

Comment:
This will be deprecated anyway, right? No need to change it, just for my understanding

Reply:
Very likely yes.